ambry-api/src/main/java/com/github/ambry/config/HelixAccountServiceConfig.java

```diff
@@ -99,6 +100,20 @@
   @Default("100")
   public final int totalNumberOfVersionToKeep;
 
+  /**
+   * The number of retry times when the update accounts fails by marking delete_in_progress container status to inactive;
+   */
+  @Config(RETRY_COUNT)
+  @Default("10")
+  public final int retryCount;
+
+  /**
+   * The sleep time between each retry action when the update accounts fails by marking delete_in_progress container status to inactive;
+   */
+  @Config(RETRY_DELAY)
+  @Default("1000")
+  public final long retryDelay;
```

Comment:
Can we add time unit into the name? Like retryDelaySeconds or retryDelayMs ?

Reply:
Sure. Updated.